### ISSUE TITLE
InvalidOperationException when importing protobuf having pattern validation

### ISSUE THREAD

**Issue by solita-simoki** [riok/Kreya#100]
**Describe the bug**
InvalidOperationException when importing protobuf definitions having [protoc-gen-validate](https://github.com/envoyproxy/protoc-gen-validate) string rule with pattern "\\d{4}". Maybe the escape sequence is not handled correctly.

For example:
```
message ExampleMessage {
    google.protobuf.StringValue status = 1 [(validate.rules) = {string:{len:7, pattern:"\\d{4}"}, message:{required:true}}];
}
```

If needed I can provide full example of protobufs.

**To Reproduce**
1. Go to Project Settings → Importers
2. Define Proto files by 'Add proto directory'
3. Click Save and Run all importers

**Expected behavior**
Successful protobuf importing.

**Environment (if possible, copy the information from the error dialog or the About menu):**
 - OS: macOS 12.5 (Apple M1)
 - Kreya Version 1.8.1 (arm64)

**Additional context**
```
{
  "kreyaVersion": "1.8.1",
  "platform": "MacIntel",
  "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko)"
}

System.InvalidOperationException: Cannot write a Varint/19 header until the String/6 data has been written; writer: ProtoBuf.ProtoWriter+StreamProtoWriter
   at ProtoBuf.Internal.ThrowHelper.ThrowInvalidOperationException(String message, Exception innerException) in /_/src/protobuf-net.Core/Internal/ThrowHelper.cs:line 55
   at ProtoBuf.ProtoWriter.State.<WriteFieldHeader>g__FailPendingField|34_0(ProtoWriter writer, Int32 fieldNumber, WireType wireType) in /_/src/protobuf-net.Core/ProtoWriter.State.WriteMethods.cs:line 89
   at ProtoBuf.ProtoWriter.State.WriteFieldHeader(Int32 fieldNumber, WireType wireType) in /_/src/protobuf-net.Core/ProtoWriter.State.WriteMethods.cs:line 84
   at Google.Protobuf.Reflection.FileDescriptorProto.AppendOption(FileDescriptorProto file, State& state, ParserContext ctx, String extendee, OptionHive option, Boolean resolveOnly, Int32 depth, Boolean messageSet) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1696
   at Google.Protobuf.Reflection.FileDescriptorProto.AppendOptions(FileDescriptorProto file, State& state, ParserContext ctx, String extendee, List`1 options, Boolean resolveOnly, Int32 depth, Boolean messageSet) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1485
   at Google.Protobuf.Reflection.FileDescriptorProto.AppendOption(FileDescriptorProto file, State& state, ParserContext ctx, String extendee, OptionHive option, Boolean resolveOnly, Int32 depth, Boolean messageSet) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1564
   at Google.Protobuf.Reflection.FileDescriptorProto.AppendOptions(FileDescriptorProto file, State& state, ParserContext ctx, String extendee, List`1 options, Boolean resolveOnly, Int32 depth, Boolean messageSet) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1485
   at Google.Protobuf.Reflection.FileDescriptorProto.AppendOption(FileDescriptorProto file, State& state, ParserContext ctx, String extendee, OptionHive option, Boolean resolveOnly, Int32 depth, Boolean messageSet) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1564
   at Google.Protobuf.Reflection.FileDescriptorProto.AppendOptions(FileDescriptorProto file, State& state, ParserContext ctx, String extendee, List`1 options, Boolean resolveOnly, Int32 depth, Boolean messageSet) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1485
   at Google.Protobuf.Reflection.FileDescriptorProto.ResolveOptions(ParserContext ctx, ISchemaOptions options) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1418
   at Google.Protobuf.Reflection.FileDescriptorProto.ResolveTypes(ParserContext ctx, List`1 fields, IType parent, Boolean options) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1219
   at Google.Protobuf.Reflection.FileDescriptorProto.ResolveTypes(ParserContext ctx, DescriptorProto type, Boolean options) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1310
   at Google.Protobuf.Reflection.FileDescriptorProto.ResolveTypes(ParserContext ctx, Boolean options) in /_/src/protobuf-net.Reflection/Parsers.cs:line 1339
   at Google.Protobuf.Reflection.FileDescriptorSet.Process() in /_/src/protobuf-net.Reflection/Parsers.cs:line 267
   at Kreya.Grpc.Core.Importer.ProtoFile.ProtoFileImporter.<>c__DisplayClass11_0.<ImportFromFiles>b__1()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Kreya.Grpc.Core.Importer.ProtoFile.ProtoFileImporter.ImportFromFiles(IProject project, ProtoFileImporterOptions options)
   at Kreya.Grpc.Core.Importer.ProtoFile.ProtoFileImporter.ImportFileDescriptors(IProject project, ProtoFileImporterOptions options, Boolean allowCache, String cacheKey)
   at Kreya.Grpc.Core.Importer.BaseGrpcImporter`1.Import(IImportStream`1 importStream, Boolean allowCache)
   at Kreya.Core.Importer.ImporterService.RunSingleImporter(ImportStream importStream, Boolean allowCache, IReadOnlyDictionary`2 operationsByInvoker)
   at Kreya.Core.Importer.ImporterService.RunImporters(Boolean allowCache)
   at SpiderEye.Bridge.ApiMethod.InvokeAsync(Object instance, Object parameter)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.ResolveCall(String id, String parameters)
```

**Comment by CommonGuy** [comment]
This is the same bug as #43. We use https://github.com/protobuf-net/protobuf-net to parse the .proto files, which does not handle the escape sequences correctly. The bug in protobuf-net has been fixed recently, but is not yet released. We hope to ship the bugfix with the next Kreya release.

Currently, there are three possible workarounds:

- Modify the .proto file manually to remove all options or escape sequences, since Kreya does not care about them
- Instead of importing the .proto files, import a [file descriptor set](https://kreya.app/docs/importers/#import-via-file-descriptor-sets)
- Import the proto definition via [server reflection](https://kreya.app/docs/importers/#import-via-grpc-server-reflection)

**Comment by CommonGuy** [comment]
We just released Kreya 1.9, which contains a fix for this bug.
